site/_contributors/01-antonin-bas.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
I don't think that we actually display these "contributors" profile and that's probably best this way. Remove the files?